prow/config/config.go

```diff
@@ -2529,10 +2529,36 @@ func StringsToOrgRepos(vs []string) []OrgRepo {
 
 // mergeFrom merges two prow configs. It must be called _before_ doing any
 // defaulting.
+// If you extend this, please also extend HasConfigFor accordingly.
 func (pc *ProwConfig) mergeFrom(additional *ProwConfig) error {
 	emptyReference := &ProwConfig{BranchProtection: additional.BranchProtection}
 	if diff := cmp.Diff(additional, emptyReference); diff != "" {
 		return fmt.Errorf("only 'branch-protection' may be set via additional config, all other fields have no merging logic yet. Diff: %s", diff)
 	}
 	return pc.BranchProtection.merge(&additional.BranchProtection)
 }
+
+func (pc *ProwConfig) HasConfigFor() (global bool, orgs sets.String, repos sets.String) {
```

Comment:
Do we maybe want this as an interface so this particular method can iterate over sub-structs that implement it instead of growing this one call-site to handle all types of ways that the config can be global or apply to an org, etc?

Reply:
This doesn't need to be an interface in oder to allow this to iterate over its substructs. It is important that callers get one method to call, everything else is an implementation detail in the config package.

Reply:
Sure, I was asking for how we wanted to standardize that iteration, about the implementation detail - I'll hopefully help you write these for all sub-structs once the plumbing is in :)

Reply:
Yeah I got that, but I don't think we add any value by adding an interface, but will limit how the substructs implement this.